src/main/groovy/org/mockito/release/internal/gradle/util/ReleaseNotesSerializer.java

```diff
@@ -0,0 +1,10 @@
+package org.mockito.release.internal.gradle.util;
+
+import org.mockito.release.notes.model.ReleaseNotesData;
+
+import java.util.Collection;
+
+public interface ReleaseNotesSerializer {
```

Comment:
I suspect that we don't need the interface. I use interfaces for public APIs of packages, while keeping the implementation class package-protected.

In your case DefaultReleaseNotesSerializer is public internal class used directly by the clients. The interface is not really needed and removing it will make the code slightly cleaner.